README.md

```diff
@@ -102,7 +102,55 @@ Contributing to Chimoney's Hacktoberfest is easy! Follow these steps:
 
 5. Create a pull request explaining your contribution.
 
-## 🧿 Issues
+## Onboarding enhancement
+
```

Comment:
Should there be a gitpod icon/button which when clicked auto sets up this project on gitpod? That’s what I’ve seen other projects do. We don’t want to teach people how to use gitpod but instead include the Gitpod and spaces but and a few lines of instructions 

maybe link out to official docs of both pod and spaces